### PR TITLE
Add Tonberry

### PR DESCRIPTION
**Pull request by Ayehavgunne**
**Checklist**

- [ x] This project is explicitly related to ASGI.
- [ x] The new list entry contains a project name, URL and description.
- [ x] Alphabetical order is preserved across entries and sections.

**What is this project?**

[Tonberry](https://ayehavgunne.github.io/Tonberry/) - Tonberry is an ASGI framework that takes a class based approach to routing. Influenced by CherryPy but made compatible with asyncio and WebSockets.

**Do you know about other similar projects?**

Yes

**If so, how is this one different?**

While Quart is following Flask, Tonberry takes it's influence from CherryPy, though it doesn't try to replicate the API exactly. The additions of type hint based content-type setting for response headers, automatic deserialization of requests/serialization of responses and support for nested dataclasses I think makes it stand apart from Quart and other ASGI frameworks.

---

_Anyone who agrees with this pull request can add a 👍._

### CODE REVIEW

**Comment by florimondmanca** [comment]
Hi there, thanks for your submission!

For now I'm not really inclined towards including these projects into this list. We already have several web frameworks here, most of them very popular already, so while Tonberry/Qactuar definitely look interesting, I'd be tempted to wait for these to gain a tad more adoption before adding them here.

Going to leave this open a bit longer to see if any votes chime in, though. :-)

**Comment by florimondmanca** [comment]
Heya!

After letting this decay a bit, I think Tonberry would be a nice addition to the list.

Has Qactuar been discontinued? Currently the link returns a 404 and the repo doesn't exist anymore either

**Comment by Ayehavgunne** [comment]
Yeah, Qactuar is private for now while I get authorization from my employer to make it open source. They are fine with that but we haven't gone through the paperwork yet.

I have been focused on other projects so when I get back to it I need to do some significant refactoring also.

Thanks for taking another look at this!

File: README.md [hunk 1]
```diff
@@ -145,6 +146,7 @@ _Web servers for ASGI applications._
 
 - [Daphne](http://github.com/django/daphne) - An HTTP, HTTP2 and WebSocket protocol server for ASGI, developed to power Django Channels.
 - [Hypercorn](https://pgjones.gitlab.io/hypercorn/index.html) - An ASGI server based on the sans-io hyper, h11, h2, and wsproto libraries. Supports HTTP/1, HTTP/2, WebSockets, ASGI 2.0 and ASGI 3.0. Compatible with asyncio, uvloop and trio worker types.
+- [Qactuar](https://ayehavgunne.github.io/Qactuar/) - An ASGI server with the aim of providing multiple request handling models to choose from. Still early in development. It currently can provide a post-fork model for handling each request in it's own process in cases where a strict isolation of memory is desired.
```
Comment (florimondmanca) [inline README.md:149]:
```suggestion
```

**Comment by Ayehavgunne** [comment]
Yeah. I will remove Qactuar from the PR later today after work.

**Pull request merged**